client/src/components/train/ModelTable.tsx

```diff
@@ -122,4 +136,21 @@ const ModelTable: React.FC = () => {
   );
 };
 
+type Status = {
+  status: TrainingStatus;
+};
+
+const ModelStatusIndicator: React.FC<Status> = ({status}) => {
+  switch (status) {
+    case TrainingStatus.Waiting:
+      return <Target color="green" />;
+    case TrainingStatus.Training:
+      return <Target color="#ccc" />;
```

Comment:
Perhaps the Feather `Loader` icon would be more appropriate than `Target` for training? https://feathericons.com/?query=loader